calico/felix/refcount.py

```diff
@@ -108,6 +116,9 @@ def decref(self, object_id):
         Return a reference and garbage-collect the backing actor if it is no
         longer referenced elsewhere.
         """
+        # MD4 RefHelper implies that this method can't be called while the object is
+        # starting - if this is true, it needs to be in this docstring, if not then
+        # RefHelper can be simplified a bunch.
```

Comment:
It is valid to decref while an object is starting.  The reason that RefHelper doesn't do that is to simplify the code in RefHelper.  Without that, RefHelper needs to account for potential in-flight callbacks from  objects that it started but then stopped again before receiving the callbacks.

I wrote the code that way at first but it was ugly and Peter suggested the approach in RefHelper.